authors/apps/comments/views.py

```diff
@@ -0,0 +1,150 @@
+from .renderers import CommentRenderer, ReplyRenderer
+from django.shortcuts import get_object_or_404
+from rest_framework import generics, status
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from .serializers import CommentSerializer, ReplySerializer
+from ..profiles.models import Profile
+from .models import Comment, Reply
+from ..articles.models import Article
+from rest_framework.response import Response
+
+
+class CommentsAPIView(generics.GenericAPIView):
```

Comment:
@sebbss, be specific with the `generic view you trying to use` For example for `post` and `get` , you can use ListCreateAPIView.
More details on https://www.django-rest-framework.org/api-guide/generic-views